src/features/homePage/HomePage.tsx

```diff
@@ -1,5 +1,115 @@
+import * as Yup from 'yup';
+import { ErrorMessage, Field, Form, Formik } from 'formik';
+import { useTranslation } from 'react-i18next';
+
+import group from '../../assets/groupLogo.png';
+import { mock } from '../products/productSeries/mockSeries';
```

Comment:
Could You please add mock file with more descriptive name to assets: ?
![image](https://user-images.githubusercontent.com/66994199/167703055-4ab1cd77-130a-4474-b9a0-950ab90055bc.png)
It will be a lot easier to clean it later.